Remote tests fail with: `User does not belong to group`

The remote tests for qasm and qir fail with a `User does not belong to group` error. The failing tests are:

```FAILED integration/backend_test.py::test_submission_with_group[Language.QASM-H1-1SC] - pytket.extensions.quantinuum.backends.api_wrappers.QuantinuumAPIError: HTTP error submitting job, {'code': 64, 'message': 'User does not belong to group'}
FAILED integration/backend_test.py::test_submission_with_group[Language.QASM-H1-2SC] - pytket.extensions.quantinuum.backends.api_wrappers.QuantinuumAPIError: HTTP error submitting job, {'code': 64, 'message': 'User does not belong to group'}
FAILED integration/backend_test.py::test_submission_with_group[Language.QASM-H2-1SC] - pytket.extensions.quantinuum.backends.api_wrappers.QuantinuumAPIError: HTTP error submitting job, {'code': 64, 'message': 'User does not belong to group'}
FAILED integration/backend_test.py::test_submission_with_group[Language.QIR-H1-1SC] - pytket.extensions.quantinuum.backends.api_wrappers.QuantinuumAPIError: HTTP error submitting job, {'code': 64, 'message': 'User does not belong to group'}
FAILED integration/backend_test.py::test_submission_with_group[Language.QIR-H1-2SC] - pytket.extensions.quantinuum.backends.api_wrappers.QuantinuumAPIError: HTTP error submitting job, {'code': 64, 'message': 'User does not belong to group'}
FAILED integration/backend_test.py::test_submission_with_group[Language.QIR-H2-1SC] - pytket.extensions.quantinuum.backends.api_wrappers.QuantinuumAPIError: HTTP error submitting job, {'code': 64, 'message': 'User does not belong to group'}

--- a/pytket/extensions/quantinuum/backends/quantinuum.py
+++ b/pytket/extensions/quantinuum/backends/quantinuum.py
@@ -587,6 +587,8 @@ class QuantinuumBackend(Backend):
         if s == "":
             return None
         bits = json.loads(s)
+        if bits is None:
+            return None
         assert all(isinstance(name, str) and isinstance(idx, int) for name, idx in bits)
         return bits
 

--- a/tests/integration/backend_test.py
+++ b/tests/integration/backend_test.py
@@ -552,7 +552,7 @@ def test_submission_with_group(
     c.measure_all()
     c = b.get_compiled_circuit(c)
     n_shots = 10
-    shots = b.run_circuit(c, n_shots=n_shots, group="DEFAULT").get_shots()  # type: ignore
+    shots = b.run_circuit(c, n_shots=n_shots, group="Default - UK").get_shots()  # type: ignore
     assert all(q[0] == q[1] for q in shots)
 
 

--- a/tests/integration/backend_test.py
+++ b/tests/integration/backend_test.py
@@ -770,7 +770,7 @@ def test_allow_2q_gate_rebase(authenticated_quum_backend: QuantinuumBackend) -> 
     r = b.get_results(h)[0]
     shots = r.get_shots()
     assert len(shots) == 1
-    assert len(shots[0]) == 1
+    assert len(shots[0]) == 2
 
 
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)

